Cargo.toml

```diff
@@ -30,6 +30,12 @@ std = ["alloc", "serde?/std"]
 alloc = ["serde?/alloc"]
 derive = ["bincode_derive"]
 
+## !! Not for public use !!
+# features to check again in CI
+ci = ["std", "alloc", "derive", "serde"]

```

Comment:
I'd rather we just use the feature directly instead of having a convenience feature just for CI